src/lib/y2storage/filesystems/base.rb

```diff
@@ -65,6 +82,17 @@ def in_network?
       def types_for_is
         super << :filesystem
       end
+
+      FREE_SPACE_FALLBACK = DiskSize.new(0)
+      def detect_free_space
```

Comment:
NP: Probably the name does not fit very well. This method is simply proposing a fallback value, why not to call it `free_space_fallback` ? Only an idea.